content/applications/general/email_communication/email_servers.rst

```diff
@@ -22,16 +37,16 @@ The following sections below contain information on how to integrate an external
 Odoo.
 
 .. warning::
-   If no one in the company is used to managing email servers, Odoo Online and Odoo.sh are strongly
-   recommended. Those Odoo hosting types' email systems work instantly and are monitored by
-   professionals. Nevertheless, a company can use their own email server if they want to manage the
-   email server's reputation themselves.
+   If no one in the company is employed to manage email servers, Odoo Online and Odoo.sh are
+   strongly recommended. In those Odoo hosting types' email systems work instantly and are monitored
+   by professionals. Nevertheless, a company can use their own email server if they want to manage
+   the email server's reputation themselves.
```

Comment:
This would imply to do the DKIM configuration according to their outgoing mail server ;)

Reply:
Yes and I've linked the doc for email DNS. Good call.